chsdi/lib/files_handler.py

```diff
@@ -39,7 +39,10 @@ def get_item(self, admin_id):
         item = None
         try:
             item = self.table.get_item(Key={'adminId': str(admin_id)}).get('Item', None)
-        except Exception:
+        except Exception as e:
+            log.error("----------------------------------------")
```

Comment:
Cleanup?

Reply:
It's still a work in progress (I should provide a test link), I'm keeping some logs here and there. But I'll cleanup most of the logs before going live